frontend/src/utilities/NavData.tsx

```diff
@@ -97,31 +97,47 @@ const useHomeNav = (): NavDataItem[] =>
 
 const useDSProjectsNav = (): NavDataItem[] =>
   useAreaCheck(SupportedArea.DS_PROJECTS_VIEW, [
-    { id: 'dsg', label: 'Data Science Projects', href: '/projects' },
+    { id: 'dsg', label: 'Data science projects', href: '/projects' },
   ]);
 
+const useModelsNav = (): NavDataItem[] => {
+  const isFineTuningAvailable = useIsAreaAvailable(SupportedArea.FINE_TUNING).status;
+
+  const modelCustomizationNavChild = isFineTuningAvailable
+    ? [{ id: 'modelCustomization', label: 'Model customization', href: '/modelCustomization' }]
+    : [];
+
+  return [
+    {
+      id: 'models',
+      group: { id: 'models', title: 'Models' },
+      children: [
+        ...useAreaCheck(SupportedArea.MODEL_CATALOG, [
+          { id: 'modelCatalog', label: 'Model catalog', href: '/modelCatalog' },
+        ]),
+        ...useAreaCheck(SupportedArea.MODEL_REGISTRY, [
+          { id: 'modelRegistry', label: 'Model registry', href: '/modelRegistry' },
+        ]),
+        ...useAreaCheck(SupportedArea.MODEL_SERVING, [
+          { id: 'modelServing', label: 'Model deployments', href: '/modelServing' },
+        ]),
+        ...modelCustomizationNavChild,
```

Comment:
why not just remove the consts above and

```suggestion
        ...useAreaCheck(SupportedArea.FINE_TUNING, [
          { id: 'modelCustomization', label: 'Model customization', href: '/modelCustomization' },
        ]),
```

Reply:
regardless of what I set in OdhDashboardConfig with regards to models, I cannot make the models tab disappear from the nav pane right under Data Science Projects. Is this intended?
See slack odh-io.slack.com support entry of mine